### ISSUE TITLE
Silent miner not working

### ISSUE THREAD

**Issue by Dawnology** [UnamSanctam/SilentCryptoMiner#1746]
I'm using your example setting but when i run it (test.exe in VM ofc), it does not show anything in test-checker.exe file
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/a1bb510e-4151-499d-8038-0d6d93534ad4)

**Comment by Dawnology** [comment]
> I'm using your example setting but when i run it (test.exe in VM ofc), it does not show anything in test-checker.exe file ![image](https://user-images.githubusercontent.com/104293532/259948504-a1bb510e-4151-499d-8038-0d6d93534ad4.png)

Im using setting Full

**Comment by UnamSanctam** [comment]
Did you change any settings at all?

**Comment by Dawnology** [comment]
i removed etc and replaced the wallet address of xmr in the miner

**Comment by UnamSanctam** [comment]
If you load the example settings and do not change any of the settings at all then does it work? Since then we can know if it's due to the changes or not.

**Comment by Dawnology** [comment]
i ran it but it doesn't work either
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/1c974575-53bf-4f03-a663-3b26b5b6914c)

**Comment by Dawnology** [comment]
nope

**Comment by Dawnology** [comment]
you mean the startup entry inside the Task Scheduler in the VM?

**Comment by UnamSanctam** [comment]
Yes where you ran the miner, to know whether it was able to create the startup Task.

**Comment by Dawnology** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/f1d883cb-0a82-44b7-ad79-36f15cfbd74e)
im sorry but the VM has a problem, I'll answer when its ok. Seriously I don't know what going on in my VM rn

**Comment by Dawnology** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/cc0ac1ea-0613-496f-9872-9235fac99542)
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/7c3359f4-8722-4a12-b833-c0ba77aadab7)
i've restart the VM and this is the startup entry

**Comment by Dawnology** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/417d1cbd-42c7-4cd6-becc-e5d714836776)

**Comment by UnamSanctam** [comment]
Alright so that is there too, and if you run the checker (the specific checker for the installed miner) it still says that it's not running?

**Comment by UnamSanctam** [comment]
And that's the checker for the specific miner that's installed?

**Comment by Dawnology** [comment]
umm what do you mean?

**Comment by UnamSanctam** [comment]
I mean is that the specific checker that was built together with the miner that's currently installed? Since a checker will only work for the same miner that it was built with and no other miners.

**Comment by UnamSanctam** [comment]
Do you have any other VMs or computers to test it on? I just tested it on my VMs (and computer) and it still works on all of them for me, since it's able to install it correctly for you then there might be some issue with injecting the watchdog and miners. If you have any other VMs or computers to test then we can see if it's just due to something specific on that VM.

**Comment by Dawnology** [comment]
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/104293532/15e46ca1-4e11-4e73-b8dc-c7d47bb8fecd)
i tested on 3 other VM and it is still the same as above

**Comment by Dawnology** [comment]
Its 3 AM at me rn, so I'm gonna answer tomorrow

**Comment by Dawnology** [comment]
Do you have any solution to solve this?

**Comment by Dawnology** [comment]
nope

**Comment by opget** [comment]
I am experiencing a similar problem tested on different virtual machines, when you build a build without implementing in the system the miner starts, but when you add options from "options" the miner does not work. And initially everything worked! When I built 3-4 builds I faced the same problem. I tried reinstalling VMs clean vin 10 and build your project again.

**Comment by opget** [comment]
> Вот майнер (пример настроек lite), который я только что протестировал на чистой Windows 10 и чистой Windows 11, который работал у меня, если вы хотите это проверить: [MinerTest.zip](https://github.com/UnamSanctam/SilentCryptoMiner/files/12322676/MinerTest.zip)

This build will not run in a clean virtualized win 10

**Comment by UnamSanctam** [comment]
>I am experiencing a similar problem tested on different virtual machines, when you build a build without implementing in the system the miner starts, but when you add options from "options" the miner does not work. And initially everything worked! When I built 3-4 builds I faced the same problem. I tried reinstalling VMs clean vin 10 and build your project again. But it didn't work for me, help please

Which "Option" breaks it for you? Can you try creating multiple builds with each of the different "Options" you use enabled (only one option per build) so that you can see which build it breaks for?

>This build does not work in a clean virtualized Win 10.

Hmm, I can't recreate the issue. I just tried creating a new fully clean Windows 10 VM (and Windows 11) but it works for me without issue there with the example settings at least.

**Comment by opget** [comment]
> > У меня аналогичная проблема проверена на разных виртуалках, при сборке сборки без внедрения в систему майнер запускается, но при добавлении опций из "options" майнер не работает. И изначально все работало! Когда я построил 3-4 сборки, я столкнулся с той же проблемой. Я попытался переустановить виртуальные машины с чистой вин 10 и снова собрать ваш проект. Но у меня не получилось, помогите пожалуйста
> 
> Какой «Вариант» ломает его для вас? Можете ли вы попробовать создать несколько сборок с включенными различными «параметрами», которые вы используете (только один параметр на сборку), чтобы вы могли видеть, для какой сборки она прерывается?
> 
> > Эта сборка не работает в чистой виртуализированной Win 10.
> 
> Хм, я не могу воссоздать проблему. Я только что попытался создать новую полностью чистую виртуальную машину Windows 10 (и Windows 11), но у меня она работает без проблем, по крайней мере, с настройками примера.

I emailed you, but I didn't get a reply. I have a good offer for you. Can I contact you by PM?

**Comment by UnamSanctam** [comment]
>I emailed you, but I didn't get a reply. I have a good offer for you. Can I contact you by PM?

Sure, I might have missed it since I get quite many emails every day and sometimes I accidentally click the notification (so that it becomes read) and thus miss it. I'm making the finishing touches for the next miner version at the moment.

**Comment by opget** [comment]
> > У меня аналогичная проблема проверена на разных виртуалках, при сборке сборки без внедрения в систему майнер запускается, но при добавлении опций из "options" майнер не работает. И изначально все работало! Когда я построил 3-4 сборки, я столкнулся с той же проблемой. Я попытался переустановить виртуальные машины с чистой вин 10 и снова собрать ваш проект. Но у меня не получилось, помогите пожалуйста
> 
> Какой «Вариант» ломает его для вас? Можете ли вы попробовать создать несколько сборок с включенными различными «параметрами», которые вы используете (только один параметр на сборку), чтобы вы могли видеть, для какой сборки она прерывается?
> 
> > Эта сборка не работает в чистой виртуализированной Win 10.
> 
> Хм, я не могу воссоздать проблему. Я только что попытался создать новую полностью чистую виртуальную машину Windows 10 (и Windows 11), но у меня она работает без проблем, по крайней мере, с настройками примера.

Problem solved, the problem was that I did not notice the startup timeout it was actually set to 1000. In a small frame it looked like 100 :)

**Comment by UnamSanctam** [comment]
>Problem solved, the problem was that I did not notice the startup timeout it was actually set to 1000. In a small frame it looked like 100 :)

Aha, alright. Good job finding the problem.

**Comment by opget** [comment]
> > Я написал вам по электронной почте, но не получил ответа. У меня есть для вас хорошее предложение. Могу ли я связаться с вами в ПМ?
> 
> Конечно, я мог пропустить это, так как я получаю довольно много писем каждый день, и иногда я случайно нажимаю на уведомление (чтобы оно стало прочитанным) и, таким образом, пропускаю его. В данный момент я делаю последние штрихи для следующей версии майнера.

I'll try my luck again
